src/test/java/io/xstefank/wildfly/bot/PRRuleDirectoryHitTest.java

```diff
@@ -129,41 +129,73 @@ void testDirectoriesMentionsNoHitInDiff() throws IOException {
             """;
 
         given().github(mocks -> {
-                    mocks.configFile(CONFIG_FILE_NAME).fromString(wildflyConfigFile);
-                    MockedGHPullRequestProcessor.processPullRequestMock(mocks.pullRequest(gitHubJson.id()),
-                            mockFileDetails(), MockedGHPullRequestProcessor.mockEmptyComments());
-                })
-                .when().payloadFromString(gitHubJson.jsonString())
-                .event(GHEvent.PULL_REQUEST)
-                .then().github(mocks -> {
-                    GHPullRequest mockedPR = mocks.pullRequest(gitHubJson.id());
-                    Mockito.verify(mockedPR, Mockito.times(2)).listFiles();
-                    Mockito.verify(mockedPR, Mockito.times(1)).listComments();
-                });
+                mocks.configFile(CONFIG_FILE_NAME).fromString(wildflyConfigFile);
+                MockedGHPullRequestProcessor.processPullRequestMock(mocks.pullRequest(gitHubJson.id()),
+                    mockFileDetails(), MockedGHPullRequestProcessor.mockEmptyComments());
+            })
+            .when().payloadFromString(gitHubJson.jsonString())
+            .event(GHEvent.PULL_REQUEST)
+            .then().github(mocks -> {
+                GHPullRequest mockedPR = mocks.pullRequest(gitHubJson.id());
+                Mockito.verify(mockedPR, Mockito.times(2)).listFiles();
+                Mockito.verify(mockedPR, Mockito.times(1)).listComments();
+                Mockito.verify(mockedPR, Mockito.never()).comment("/cc @7125767235");
+            });
+    }
+
+    @Test
+    void testDirectoriesNoHitInDiffIfSubstring() throws IOException {
+        wildflyConfigFile = """
+            wildfly:
+              rules:
+                - id: "Directory Test"
+                  directories:
+                   - app
+                  notify: [7125767235]
+              format:
+                commit:
+                  enabled: false
```

Comment:
Does it have to be disabled? The #101 states that checks should be enabled by default

Reply:
no, I had different issue with it, I'll remove it